src/components/Calculator.js

```diff
@@ -0,0 +1,30 @@
+import './Calculator.css';
+
+const Calculator = () => (
+  <div className="container">
+    <div className="result-display">0</div>
+    <div className="calc-buttons">
+      <button className="calc-button" type="button">AC</button>
+      <button className="calc-button" type="button"> +/- </button>
+      <button className="calc-button" type="button"> % </button>
+      <button className="calc-button-right" type="button"> ÷ </button>
+      <button className="calc-button" type="button"> 7 </button>
+      <button className="calc-button" type="button"> 8 </button>
+      <button className="calc-button" type="button"> 9 </button>
+      <button className="calc-button-right" type="button"> x </button>
+      <button className="calc-button" type="button"> 4 </button>
+      <button className="calc-button" type="button"> 5 </button>
+      <button className="calc-button" type="button"> 6 </button>
+      <button className="calc-button-right" type="button"> - </button>
+      <button className="calc-button" type="button"> 1 </button>
+      <button className="calc-button" type="button"> 2 </button>
+      <button className="calc-button" type="button"> 3 </button>
+      <button className="calc-button-right" type="button"> + </button>
+      <button className="calc-button" id="zero" type="button"> 0 </button>
+      <button className="calc-button" type="button"> . </button>
+      <button className="calc-button-right" type="button"> = </button>
+    </div>
+  </div>
+);
```

Comment:
- [ ] Please form the project requirement, you should make sure that your `Calculator.js` have at least 1 child component :pray: You can read about child and parent components [here](https://bobbyhadz.com/blog/react-pass-data-from-child-to-parent)

Reply:
@Owusu-Desmond Thanks for the review 🤝 . I will do it and submit it again 💯 .